### PR TITLE
[16.0][MIG] sale_margin_sync: Migration to version 16.0

### PR DESCRIPTION
**Pull request by Yadier-Tecnativa**
@Tecnativa TT42429

### CODE REVIEW

File: sale_margin_sync/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale margin sync",
+    "summary": "Recompute sale margin when stock move cost price is changed",
+    "version": "16.0.1.0.1",
```
Comment (pedrobaeza) [inline sale_margin_sync/__manifest__.py]:
```suggestion
    "version": "16.0.1.0.0",
```

Comment (Yadier-Tecnativa) [reply sale_margin_sync/__manifest__.py]: Changed

**Review by pedrobaeza — APPROVED** [review]

**Review by sergio-teruel — APPROVED** [review]

**Comment by sergio-teruel** [comment]
ping @rafaelbn

**Comment by sergio-teruel** [comment]
Done!
@juangomezzurita

**Pull request merged**